modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -10,21 +10,95 @@ image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
-When you deploy an application to CloudHub, you have access to the validated OS updates, security patches, and bug fixes that MuleSoft provides regularly.
+Because CloudHub is an integration platform as a service (iPaaS), 
+the platform manages patching of the operating system and Mule runtime engine,
+enabling you to focus on developing and updating your applications.
 
-You can keep your CloudHub applications up to date by manually updating them at your convenience.
-If you don't update apps manually, MuleSoft applies the updates automatically.
+CloudHub applies security patches as needed to ensure that your application is secure. 
+Once per month, CloudHub updates Mule to maintain the stability of your application.
 
-When a new runtime version is available for your CloudHub application, the *Applications* page displays a banner and the *Update Available* tab lists the apps that require updates:
+Patch updates use blue-green deployment, which ensures zero downtime.
 
+== Security Updates 
+
+MuleSoft regularly runs scans to identify security vulnerabilities in Mule runtime engine, JVM,
+and the underlying operating system, and then automatically applies security patches based on the following SLA:
+
+[%header,cols="20,25,40"]
+|===
+|Severity Level | Severity Definition | Patch Applied Within 
+|P0 | Critical | 7 days
+|P1 | High | 30 days
+|===
+
+
+== Monthly Date Patch Updates 
+
+Once per month, MuleSoft releases a _date patch_ update, 
+which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
+The monthly date patch update might include security patches if the scheduled date is within the security SLA.
+
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.2.2 - 05-04-2020`
+
+The monthly patches only update applications to the latest date patch for the patch version.
+They do not change the patch version number.
+
+
+=== Update Schedule 
+
+The schedule for date patch updates is:
+
+* In the first full week of the month, MuleSoft releases the date patch.
+* In the third week of the month, MuleSoft automatically applies the date patch that was released in the first week of the month.
```

Comment:
Will anyone question what the third week is, if the month begins with a partial week and we just referred to the "first full week"?

Reply:
Should we say "third full week"?